mhctools/cli/args.py

```diff
@@ -180,4 +185,7 @@ def mhc_binding_predictor_from_args(args):
     if args.mhc_predictor_path:
         kwargs["program_name"] = args.mhc_predictor_path
 
+    if args.do_not_raise_on_error:
+        kwargs["raise_on_error"] = False
```

Comment:
will this result in an error if the mhc_class is not one of the IEDB predictors (i.e. one where the constructor doesn't take a raise_on_error kwarg?) If so I think might be slightly clearer to update the help for the commandline option above to replace "Only applies to IEDB predictors" to something like "This may only be specified for the IEDB predictors".

Reply:
good point! It did, and I just fixed it to be ignored and log a warning